controllers/operatorscaler_controller.go

```diff
@@ -174,6 +180,52 @@ func (r *OperatorScalerReconciler) isOdfDependenciesCsvReady(ctx context.Context
 	return nil
 }
 
+func (r *OperatorScalerReconciler) reconcileMetrics(ctx context.Context, logger logr.Logger) error {
+	logger.Info("entering reconcileMetrics")
+
+	var returnErr error
```

Comment:
1. Can this please be moved to the area where it is used? direct above the for loop? 
2. This is not the only error var the might get returned so returnErr does not fit as a name, maybe aggrError is more fitting

Reply:
Moved it.